src/main.py

```diff
@@ -0,0 +1,47 @@
+import src.controllers as contr
+import src.pre_processing as pp
+import yfinance as yf
+import pandas as pd
+import numpy as np
+import os
+
+
+def correlation_testing():
+    tickers = pd.DataFrame(["GME", "AMC", "KOSS", "NAKD", "BBBY", "NOK", "VIX"], columns=['Ticker'])
+    sc = contr.StockController()
+    sc.init_stocks(tickers)
+    for stock_name, stock in sc.stocks.items():
+        stock.get_history()
+    corr = {stock_name: dict() for stock_name in sc.stocks.keys()}
+    norm_corr = {stock_name: dict() for stock_name in sc.stocks.keys()}
```

Comment:
Not a big deal, but seems better to me:
```suggestion
    import copy
    norm_corr = copy.deepcopy(corr)
```

Reply:
Does a deep copy here also deep copy underlying objects? Because I believe you need to write your own deep copy method as soon as there are objects within the copy. Correct me if I'm wrong :)